permutableString.js

```diff
@@ -0,0 +1,38 @@
+// https://dmitripavlutin.com/how-to-compare-objects-in-javascript/
+//Link above - compare objects, 3 subheader - don't clear 'Object.keys'...
+//Works when both of strings are the same length.
+function permutable(s1, s2) {
+    let obj1 = {}, obj2 = {};
+
+    function countSymbols(str) {
+        let obj = {}
+        for (let i = 0; i < str.length; i++) {
+            if (!(str[i] in obj)) {
+                obj[str[i]] = 1;
+            } else {
+                obj[str[i]] += 1;
+            }
+        }
+        return obj;
+    }
+
+    obj1 = countSymbols(s1);
+    obj2 = countSymbols(s2);
+    
+    countSymbols(s1, obj1);
+    countSymbols(s2, obj2);
+
+
+    const keys1 = Object.keys(obj1);
```

Comment:
Please refactor object equality comparison in a separate function, `isEqual(obj1, obj2)`, which returns `true` if the objects are equal and `false` otherwise.